internal/rootcoord/quota_center.go

```diff
@@ -145,6 +145,13 @@ func (q *QuotaCenter) run() {
 			log.Info("QuotaCenter exit")
 			return
 		case <-ticker.C:
+			// syncMetrics may cost several seconds when qc/dc is offline, and with golang's random select policy
+			// quotaCenter' Stop may block here for minutes. which also affect rootcood's stop
+			_, closed := <-q.stopChan
```

Comment:
Please see line L177, stopChan is not closed but have a singal
So Signal will never to `true` here